store/tikv/prewrite.go

```diff
@@ -156,6 +170,21 @@ func (action actionPrewrite) handleSingleBatch(c *twoPhaseCommitter, bo *Backoff
 					c.mu.Unlock()
 				}
 			}
+			if c.isOnePC() {
+				if prewriteResp.OnePcCommitTs == 0 {
+					logutil.Logger(bo.ctx).Warn("1pc failed and fallbacks to normal commit procedure",
+						zap.Uint64("satrTS", c.startTS))
+					tikvOnePCTxnCounterFallback.Inc()
+				} else {
+					c.mu.Lock()
+					if c.onePCCommitTS != 0 {
+						logutil.Logger(bo.ctx).Fatal("one pc happened multiple times",
```

Comment:
Is it possible?

Reply:
Only when we send 1PC request to multiple regions?

Reply:
It's not possible as long as there's no bug (so it's "Fatal" here)